cloudfront-dist/deployment/preview.env

```diff
@@ -1,13 +1,13 @@
 AWS_DEFAULT_REGION=eu-west-1
 STACK_PREFIX_NAME=preview
-STACK_VERSION=2.0.0
+STACK_VERSION=3.0.1
```

Comment:
it's unused right ? Can't we remove it ?

Reply:
Done